network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -1196,13 +1195,15 @@ def exec(self):
                                       'No traffic is specified as forbidden.')
         return IntersectsQuery(self.config1, self.config2).exec(True)
 
-    def compute_query_output(self, query_answer, _):
-        if query_answer.bool_result:
-            query_output = f'{self.config2.name} does not forbid connections specified in {self.config1.name}:' \
-                           f'{query_answer.output_explanation}'
+    def compute_query_output(self, query_answer, cmd_line_flag=False):
+        res = not query_answer.numerical_result if cmd_line_flag else query_answer.bool_result
```

Comment:
This converts the result to bool even if required numerical result...
Can change to `query_answer.numerical_result` (without the `not`) and change the numerical result to be 1 in case of the bool result is True.

Reply:
Forbids calls the IntersectsQuery class exec method to compute the result so the same answer of the comment above is relevant here